sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -79,6 +102,39 @@ export interface AccessToken {
    * The timestamp when the access token should be refreshed, in milliseconds, UNIX epoch time.
    */
   refreshAfterTimestamp?: number;
+
+  /** Type of token - Bearer or PoP */
+  tokenType?: string;
```

Comment:
Is this not required when you create or use a token type?

Reply:
It is required and will be mentioned when we create or use token type going forward. But we wanted to keep it backwards compatible with previous versions of core

Reply:
The default value of token type of not mentioned is Bearer

Reply:
But if we are ok with not having backward compatibility with core, we can switch it to required